app/code/BodenkoVV/AskQuestion/Console/Command/UpdateProductQty.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace BodenkoVV\AskQuestion\Console\Command;
+
+use Magento\Catalog\Api\ProductRepositoryInterface;
+use Magento\CatalogInventory\Api\StockRegistryInterface;
+use Magento\CatalogInventory\Api\StockStateInterface;
+use Magento\Framework\App\State;
+use Magento\Framework\Exception\LocalizedException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Magento\Framework\App\Area;
+
+class UpdateProductQty extends \Symfony\Component\Console\Command\Command
+{
+
+    /** @var State */
+    private $state;
+
+    /** @var StockStateInterface  */
+    protected $_stockStateInterface;
+
+    /** @var Magento\CatalogInventory\Api\StockRegistryInterface */
+    protected $_stockRegistry;
+
+    /** @var ProductRepositoryInterface  */
+    public $productRepository;
+
+    /**
+     * CustomQtyUpdate constructor.
+     * @param ProductRepositoryInterface $productRepository
+     * @param StockStateInterface $stockStateInterface
+     * @param StockRegistryInterface $stockRegistry
+     * @param State $state
+     */
+    public function __construct(
+        ProductRepositoryInterface $productRepository,
+        StockStateInterface $stockStateInterface,
+        StockRegistryInterface $stockRegistry,
+        State $state
+    )
+    {
+        $this->productRepository = $productRepository;
+        $this->_stockStateInterface = $stockStateInterface;
+        $this->_stockRegistry = $stockRegistry;
+        $this->state = $state;
+        parent::__construct();
+    }
+
+    /**
+     * @param $productId
+     * @param $stockData
+     * @return bool
+     * @throws \Magento\Framework\Exception\NoSuchEntityException
+     */
+    public function updateProductStock($productId, $stockData) {
```

Comment:
Please rename $stockData to $qty, because it's not obvious what is $stockData, whether it is an object, what type of it object? If the variable is called qty or quantity - it's clear that it is a number

Reply:
+